integration-cli/docker_cli_by_digest_test.go

```diff
@@ -80,10 +77,6 @@ func (s *DockerRegistrySuite) TestPullByTagDisplaysDigest(c *check.C) {
 	testPullByTagDisplaysDigest(c)
 }
 
-func (s *DockerSchema1RegistrySuite) TestPullByTagDisplaysDigest(c *check.C) {
```

Comment:
The PR claims to be removing push, so why are these tests being removed?

Reply:
The whole schema1 test suite must be removed since that old version of the registry is no longer supported with Docker. If you can't push the image, you can't run the pull test. A new test which pulls an old version from the hub could be added, but that is fragile.

Reply:
These tests were added after a CVE fix so it seems weird to remove it without an alternative as the pull code is still remaining(probably for years) and needs to be validated.